automatic/zotero/zotero.nuspec

```diff
@@ -2,30 +2,32 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
+    <!-- == PACKAGE SPECIFIC SECTION == -->
     <id>zotero</id>
-    <title>Zotero</title>
+    <version>7.0.7</version>
+    <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/zotero</packageSourceUrl>
     <owners>chocolatey-community</owners>
-    <version>7.0.5</version>
-    <authors>Center for History and New Media, et. al.</authors>
-    <summary>Zotero [zoh-TAIR-oh] is a free, easy-to-use tool to help you collect, organize, cite, and share your research sources. </summary>
-    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@53607633ce049d5d75ac668f4408faaeced36bc3/icons/zotero.png</iconUrl>
-    <!-- Do not touch the description here in the nuspec file. Description is imported during update from the Readme.md file -->
-    <description><![CDATA[Zotero is free and open-source reference management software to manage bibliographic data and related research materials (such as PDF files). Notable features include web browser integration, online syncing, generation of in-text citations, footnotes and bibliographies, as well as integration with the word processors Microsoft Word, LibreOffice, OpenOffice.org Writer and NeoOffice.
-
-
-![screenshot](https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/d1d9f0154e370315e49f245a26a7d23e89a705cc/automatic/zotero/screenshot.png)
-]]></description>
+    <!-- == SOFTWARE SPECIFIC SECTION == -->
+    <title>Zotero</title>
+    <authors>The Corporation for Digital Scholarship and contributors</authors>
     <projectUrl>https://www.zotero.org/</projectUrl>
-    <tags>zotero references manager bibliography foss cross-platform admin</tags>
+    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@53607633ce049d5d75ac668f4408faaeced36bc3/icons/zotero.png</iconUrl>
     <copyright>2006-2011, Center for History and New Media, George Mason University, Fairfax, Virginia, USA</copyright>
     <licenseUrl>https://github.com/zotero/zotero/blob/master/COPYING</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <releaseNotes>https://www.zotero.org/support/._changelog</releaseNotes>
-    <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/zotero</packageSourceUrl>
     <projectSourceUrl>https://www.zotero.org/support/dev/source_code</projectSourceUrl>
     <docsUrl>https://www.zotero.org/support</docsUrl>
     <mailingListUrl>https://forums.zotero.org/discussions</mailingListUrl>
     <bugTrackerUrl>https://github.com/zotero/zotero/issues</bugTrackerUrl>
+    <tags>zotero references manager bibliography citations research foss cross-platform embedded</tags>
```

Comment:
since you are already touching the tags here, can you update them to follow the tag guidelines?
https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md#obligatory-tags-and-categories

Reply:
No problem.